packages/meditrak-app/app/assessment/specificQuestions/GeolocateQuestion.js

```diff
@@ -129,12 +135,12 @@ class GeolocateQuestionComponent extends React.Component {
     const { onChangeAnswer, userLocation, answer } = this.props;
     const { latitude, longitude, accuracy, errorMessage } = userLocation;
 
-    if (
-      (latitude !== null || errorMessage) &&
-      latitude !== answer.latitude &&
-      longitude !== answer.longitude &&
-      accuracy !== answer.accuracy
-    ) {
+    const locationHasChanged =
+      latitude !== answer.latitude ||
+      longitude !== answer.longitude ||
+      accuracy !== answer.accuracy;
+
+    if ((latitude !== null || errorMessage) && locationHasChanged) {
```

Comment:
I'm pretty sure that we should consider the location to have changed if it only changes in one dimension?